engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/ScriptAssertValidatorTest.java

```diff
@@ -93,34 +105,52 @@ public void emptyAliasRaisesException() throws Exception {
 		getInitializedValidator( "lang", "script", "" );
 	}
 
-	@Test(expectedExceptions = ConstraintDeclarationException.class)
```

Comment:
What's the reason to change that? I find that much more expressive personally.

Reply:
I do like the `expectedExceptions` more compared to this change. But as
now that we only try to create a `ScriptEvaluator` when `isValid` is called. We are not able to check for things like "is there an evaluator for specified language?" during initialization. As a result the exception (`ConstraintDeclarationException`) is only thrown from isValid method and it is wrapped with `ValidationException` ...

Reply:
Ok.